responsibleai_vision/requirements.txt

```diff
@@ -0,0 +1,8 @@
+numpy>=1.17.2
```

Comment:
in requirements dev you have an upper limit, do you want to be consistent?

Reply:
interesting, I see different versions all over the place here:
![image](https://github.com/microsoft/responsible-ai-toolbox/assets/24683184/502bf4cb-134d-4d5b-acb2-55ed632295a4)

Reply:
I just went ahead and removed the one in requirements-dev.txt, as we don't need a duplicate one I think...